chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -297,6 +297,23 @@
                         !node[:provisioner][:suse][:cloud_available] && node[:provisioner][:suse][:storage_available],
                       crowbar_join: "#{os_url}/crowbar_join.sh")
           end
+          template "#{node_cfg_dir}/autoyast-upgrade.xml" do
+            mode 0644
+            source "autoyast-upgrade.xml.erb"
+            owner "root"
+            group "root"
+            variables(
+                      admin_node_ip: admin_ip,
+                      web_port: web_port,
+                      packages: packages,
+                      repos: repos,
+                      timezone: timezone,
+                      target_platform_version: target_platform_version,
+                      architecture: arch,
+                      is_ses: node[:provisioner][:suse] &&
+                        !node[:provisioner][:suse][:cloud_available] && node[:provisioner][:suse][:storage_available],
```

Comment:
Metrics/LineLength: Line is too long. [118/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)